pandas/tests/series/test_reductions.py

```diff
@@ -51,29 +53,32 @@ def test_td64_sum_empty(skipna):
     assert result == pd.Timedelta(0)
 
 
-def test_td64_summation_overflow():
-    # GH#9442
-    ser = Series(pd.date_range("20130101", periods=100000, freq="H"))
-    ser[0] += pd.Timedelta("1s 1ms")
+@given(
+    st.integers(
+        min_value=0,
+        max_value=10 ** (np.finfo(np.float64).precision),
+    ).map(pd.Timedelta)
+)
+def test_td64_summation_retains_ns_precision_over_expected_range(value: pd.Timedelta):
+    result = Series(value).sum()
 
-    # mean
-    result = (ser - ser.min()).mean()
-    expected = pd.Timedelta((pd.TimedeltaIndex(ser - ser.min()).asi8 / len(ser)).sum())
+    assert result == value
 
-    # the computation is converted to float so
-    # might be some loss of precision
```

Comment:
i think itd be useful to retain this comment

Reply:
i see you moved this comment to a docstring, thanks.

For future reference, pls let the commenter hit the "resolve" button